.github/workflows/build.yml

```diff
@@ -11,13 +11,11 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        os: [ ubuntu-latest, macos-latest, windows-2019 ]
+        os: [ubuntu-latest, macos-latest, windows-2019]
 
     uses: homebridge/.github/.github/workflows/nodejs-build-and-test.yml@main
     with:
       enable_coverage: true
-      runs_on: ${{ matrix.os }}
-      install_cmd: npm ci && cd ui && npm ci
```

Comment:
What are these changes, they break the workflow run

Reply:
Adding back with 4.50.4